plugins/data-types/src/index.ts

```diff
@@ -44,19 +44,35 @@ export interface target {
 	}
 }
 
-export interface hookContex {
+export interface HookContex {
 	target: target
 }
 
+export interface TypesOptions {
+	units?: boolean
+}
+
+export interface DataTypesParams {
```

Comment:
Do we need this additional level of interfaces? code smell that it has only one member. What could be here here beside options?